draft-schlesinger-cfrg-act.md

```diff
@@ -191,10 +178,10 @@ This protocol builds upon several cryptographic primitives:
   proofs of possession. We use a variant that is privately verifiable, which
   avoids the need for pairings and makes our protocol more efficient.
 
-- **Sigma Protocols** {{ORRU-SIGMA}}: The zero-knowledge proof framework used
+- **Sigma Protocols** {{!I-D.irtf-cfrg-sigma-protocols}}: The zero-knowledge proof framework used
   for spending proofs.
 
-- **Fiat-Shamir Transform** {{ORRU-FS}}: The technique to make the interactive
+- **Fiat-Shamir Transform** {{!I-D.irtf-cfrg-fiat-shamir}}: The technique to make the interactive
```

Comment:
Consistency with ARC: see https://github.com/chris-wood/draft-arc/pull/37.

```suggestion
- **Fiat-Shamir Transform** {{!FIAT-SHAMIR=I-D.irtf-cfrg-fiat-shamir}}: The technique to make the interactive
```

Reply:
Do you have preferences on if we tag to a specific version of the fiat-shamir spec (and sigma spec), eg suffixing with `-00`? Or is it better to have this non-versioned? I'll follow suit in the ARC spec.

Reply:
I think it's best to add the version suffix.